app/facades/github_search.rb

```diff
@@ -0,0 +1,26 @@
+class GithubSearch
+  def initialize(token)
+    @token = token
+  end
+
+  def display_repos
+    get_json('repos').take(5)
+  end
+
+  def display_following
+    get_json('following').take(5)
+  end
+
+  def find_github_resource(data_type, data)
+    resource = data_type.singularize.capitalize
+    resource.constantize.new(data)
+  end
+
+  def get_json(data_type)
```

Comment:
This method can now be used for all display methods i.e. repos, following, followers